src/image_processing/command_parser/pattern_command_parser.py

```diff
@@ -0,0 +1,41 @@
+import re
+from typing import Dict, List
+
+from src.image_processing.command import Command
+from src.image_processing.command_parser.command_parser import CommandParser
+from src.image_processing.kernels.kernel_types import KernelTypes
+
+
+class PatternCommandParser(CommandParser):
+    """
+    Command parser based on patterns
+    """
+
+    def _get_patterns(self) -> Dict[KernelTypes, str]:
+        return {
+            KernelTypes.RESIZE: r"изменить размер до (\d+)x(\d+)",
+            KernelTypes.ROTATE: r"повернуть на (-?\d+) градусов",
+            KernelTypes.CONTRAST: r"изменить контрастность на (-?\d+)",
+            KernelTypes.CROP: r"обрезать до (\d+)x(\d+)",
+            KernelTypes.GRAYSCALE: r"преобразовать в черно-белый",
+            KernelTypes.INVERT: r"инвертировать цвета|инвертировать",
+            KernelTypes.BLUR: r"размытие с радиусом (\d+)",
+            KernelTypes.SHARPEN: r"повысить резкость на (-?\d+)",
```

Comment:
What about use english instead of russian?

Reply:
Agree. That was my first question of this MR.

Current implementation of `PatternCommandParser` gives the opportunity to test our idea before `AICommandParser` is implemented.

Also, patterns are strictly connected with kernel implementation. That logic must be separated.

Reply:
We can add two patterns for Russian and English scenarios in the next MR

Reply:
First of all, I'd separate patterns and kernel logic

Reply:
discussed offline